plugins/Satellites/src/gui/satellitesDialog.ui

```diff
@@ -1019,7 +1078,7 @@
               <bool>false</bool>
              </property>
              <property name="placeholderText">
-              <string notr="true"/>
+                <string>(add personal notes here)</string>
```

Comment:
For me bracketed text is OK. It is used also in other programs. Behaviour looks good. Whether it's this or only "(add notes)" is a matter of taste.

Reply:
Ok